### PR TITLE
[Feature] [spark] Support latest version of Spark 3.2.0

### LINKED ISSUE #875: support latest version of Spark 3.2.0

### PR DESCRIPTION
**Pull request by ashmeet-kandhari**
## Purpose of this pull request
Updated spark version to 3.2.0

Close #875 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] Change does not need document change, or I will submit document change to https://github.com/apache/incubator-seatunnel-website later

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -62,15 +62,15 @@
...
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
-        <scala.version>2.11.8</scala.version>
-        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.12.15</scala.version>
+        <scala.binary.version>2.12</scala.binary.version>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
-        <spark.version>2.4.0</spark.version>
-        <flink.version>1.9.0</flink.version>
```
Comment (RickyHuo) [inline pom.xml:70]: Please don't update flink version at this PR :)

**Comment by RickyHuo** [comment]
PTAL @garyelephant

File: pom.xml (hunk 1 above)
Comment (ashmeet-kandhari) [reply pom.xml:70]: reverted the change

**Comment by CalvinKirs** [comment]
> PTAL @garyelephant @CalvinKirs

HI, we'd better discuss in #875. In #875, I have the same questions as you.

**Comment by CalvinKirs** [comment]
no updates, if you have any updates, you can leave a message, and I will reopen it.

**Pull request closed without merging**